src/olympia/git/migrations/0001_initial.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 2.2.12 on 2020-04-22 11:50
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import olympia.amo.models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [('addons', '0004_auto_20191126_1712')]
+
+    operations = [
+        migrations.CreateModel(
+            name='GitExtractionEntry',
+            fields=[
+                (
+                    'id',
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name='ID',
+                    ),
+                ),
+                (
+                    'created',
+                    models.DateTimeField(
+                        blank=True,
+                        default=django.utils.timezone.now,
+                        editable=False,
+                    ),
+                ),
+                ('modified', models.DateTimeField(auto_now=True)),
+                (
+                    'addon',
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        to='addons.Addon',
+                    ),
+                ),
+                ('in_progress', models.NullBooleanField(default=None)),
+            ],
+            options={
+                'get_latest_by': 'created',
+                'abstract': False,
+                'base_manager_name': 'objects',
+                'unique_together': {('addon', 'in_progress')},
+            },
+            bases=(
+                olympia.amo.models.SearchMixin,
+                olympia.amo.models.SaveUpdateMixin,
+                models.Model,
```

Comment:
has this been run through a prettifier?  it looks non-django-migrations-y.  (which I guess is fine?  We exclude migrations from flake8 generally.  If you had to manually tinker with it can you call out your changes).

Reply:
Indeed, it's Black 🙈